File: CarsMatter.Infrastructure/Models/MsSQL/MyCar.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace CarsMatter.Infrastructure.Models.MsSQL
{
    public class MyCar
    {
        public string Id { get; set; }

        public string Name { get; set; }

        public string Brand { get; set; }

        public string Model { get; set; }

        public int Year { get; set; }

        public string UserId { get; set; }

        [JsonIgnore]
        public User User { get; set; }
    
        public ICollection<ConsumablesNote> ConsumablesNotes { get; set; }

        public ICollection<RefillNote> RefillNotes { get; set; }

        public MyCar()
        {
            this.ConsumablesNotes = new List<ConsumablesNote>();
            this.RefillNotes = new List<RefillNote>();
        }
    }
}

File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/MyCar.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MyCar {
    @SerializedName("id")
    @Expose
    public String Id;

    @SerializedName("name")
    @Expose
    public String Name;

    @SerializedName("brand")
    @Expose
    public String Brand;

    @SerializedName("model")
    @Expose
    public String Model;

    @SerializedName("year")
    @Expose
    public int Year;
}
